collection-scripts/gather_codeflare

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+source "common.sh"
+resources=("appwrappers" "quotasubtrees" "gschedulingspecs")
+
+nslist=$(get_all_namespace "$resources")
+run_mustgather "$nslist"
```

Comment:
Hmm, I though that when you define the array, you have to pass it like this `"${nslist[@]}"` otherwise only first element is passed on :thinking: 
```suggestion
run_mustgather "${nslist[@]}"
```

This applies to all other occurrences if I'm not mistaken.

Reply:
updated in new commit